community.html

```diff
@@ -149,7 +149,7 @@ <h2>RIOT Summit</h2>
                 <h2 class="mb-0">RIOT Summit 2016</h2>
                 <h3>Berlin, Germany</h3>
                 <a href="https://summit.riot-os.org/2016" class="overflow-hidden">
-                  <img src="{{ "assets/img/summits/summit2016.jpg" | relative_url }}" class="card-img" alt="">
+                  <img src="{{ "assets/img/summits/summit2016.jpg" | relative_url }}" class="card-img" alt="Card image">
```

Comment:
What is a "Card image"?

Reply:
I think this came from the Bootstrap examples. Since there's enough context around the card (title, etc), I think having a generic name is not too problematic

Reply:
hmm, i would say either "summit2016" or "Photo RIOT Summit 2016"

Reply:
> hmm, i would say either "summit2016" or "Photo RIOT Summit 2016"

I agree with @waehlisch. The alt text should provide a descriptive text what can be seen in the images. Otherwise, it can just as well not be there as just "copying" its structural purpose isn't really helpful when reading the website as a e.g. a blind person.

Reply:
ok, I'm fine with it! I update those `alt`

Reply:
done!